plugins-compat-tester-gae/src/main/webapp/index.html

```diff
@@ -286,7 +286,7 @@
                     </table>
                     {{if buildLogPath != ""}}
                     <a name="${pluginName}_${pluginVersion}_${core}" />
-                    <a href="#${pluginName}_${pluginVersion}_${core}" onclick="javascript:displayLog('${pluginName} v${pluginVersion} against ${core}', '${buildLogPath}');"><img class="terminal" src="https://github.com/jenkinsci/jenkins/raw/master/war/src/main/webapp/images/24x24/terminal.png" /></a>
+                     <a href="#${pluginName}_${pluginVersion}_${core}" onclick="javascript:displayLog('${pluginName} v${pluginVersion} against ${core}', '${buildLogPath}');"><img class="terminal" src="https://github.com/jenkinsci/jenkins/raw/master/war/src/main/webapp/images/24x24/terminal.png" /></a>
```

Comment:
I thought I had undone all the changes to this file.... ugh.  All these will definitely be squashed during merge.